### PR TITLE
make Verify also verify headers and allow numbers

### PR DESCRIPTION
**Pull request by bartekpacia**

### CODE REVIEW

File: frames.go [hunk 1]
```diff
@@ -91,18 +91,31 @@ func Assemble(header [2]byte, length byte, data []byte, checksum byte) (frame Fr
...
+// - at 3rd index: a plus sign ("+")
 //
-// - at penultimate position: have a hash sign ("#")
+// - at penultimate position: a hash sign ("#")
 //
-// - its checksum must be correct
+// - at last position: a checksum must be correct
 func Verify(frame Frame) bool {
+	first := frame[0]
+	valid1 := (first > 'A' && first < 'Z') || (first > '0' && first < '9')
```
Comment (TK-A369) [inline frames.go]: I think that there is a bug. It should be if it's greater than **or equal** `A`, and less than **or equal** `Z`. Same with digits.
This also applies line 114, where second header character is checked.

Comment (TK-A369) [reply frames.go]: In this case `A`, `Z`, `0` and `9` will be treated as errors.

**Review by TK-A369 — CHANGES_REQUESTED** [review]
I found a bug. I described it in comment.

File: frames.go (hunk 1 above)
Comment (bartekpacia) [reply frames.go]: Yes, I noticed this shortly after writing this code:) Thanks for the review anyway!

File: example/example.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/knei-knurow/frames"
+)
+
+func main() {
```
Comment (bartekpacia) [inline example/example.go:9]: This example is not really a good example, but it can be left here for now:) We can always improve it in the future.

**Review by TK-A369 — APPROVED** [review]
I haven't found any bugs. I think that this pull request is ready to merge.

**Comment by bartekpacia** [comment]
Thanks :) So I'm merging it!

**Pull request merged**